akka-http-core/src/main/scala/akka/http/scaladsl/Http.scala

```diff
@@ -753,7 +754,30 @@ trait DefaultSSLContextCreation {
   protected def system: ActorSystem
   protected def sslConfig: AkkaSSLConfig
 
-  protected def createDefaultClientHttpsContext(): HttpsConnectionContext = {
+  // --- log warnings ---
+  private[this] def log = system.log
+
+  def validateAndWarnAboutLooseSettings() = {
```

Comment:
You're right that no validation yet here – shall I rename and go over the MiMa dance?
If we'd validate ssl things it'd be here in this callback so I'd keep as is